<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Aptos;}
@font-face
	{font-family:"\@DengXian";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:8.0pt;
	margin-left:0cm;
	line-height:115%;
	font-size:12.0pt;
	font-family:"Aptos",sans-serif;}
.MsoChpDefault
	{font-size:12.0pt;
	font-family:"Aptos",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:115%;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=EN-US style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal>Good afternoon, everyone. I am very glad to present my
research today on the Classification of highly divergent protein domain
families using deep learning and hierarchical clustering.</p>

<p class=MsoNormal>My first project is to develop Domain-based Deep Learning
Models for Protein Superfamily Prediction.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>As we all know, the basic elements of proteins are 20 amino
acids. Through their different combinations,  we have millions of protein
sequences. Each of them forms a unique structure.  </p>

<p class=MsoNormal>The different protein superfamilies have unique domain
structures and topologies, and they play distinct roles in cellular processes.</p>

<p class=MsoNormal>The classification of proteins can help us understand the
function and evolution of proteins.</p>

<p class=MsoNormal>The profiles collected from MSA, such as PSSM, and contact
maps from the structure information are usually used to identify the
superfamily. However, it is tailored to the individual family, and it isn't
easy to recognize previously undefined families. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Many established models played a crucial role in protein
families/superfamilies representation, such as MSA, Consensus sequences,
Pattern matching, and Sequence motifs.</p>

<p class=MsoNormal>Additionally, Profile models exhibit better performance in
the comprehensive representation of protein features, such as the PSSM
generated by PSI-BLAST and the HMM generated by HMMER. They can be used to
explore the evolutionary information.</p>

<p class=MsoNormal>However, they all have some limitations. First, the models
are tailored to the individual family. It doesn’t work for uncovered families.
Second, it is challenging to detect highly divergent sequences using existing
methods. Besides, detecting novel proteins remains a significant challenge.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Therefore, we aim to make an AI model to predict protein superfamilies.
 Given a structure, we hope it can inform us about the superfamily to which it
belongs.</p>

<p class=MsoNormal>When we were babies, our family taught us everything we
needed to know in life. For example, the cat-dog recognition issues. They would
tell us, 'This is a dog, that is a cat, this is another type of dog.' Then we
could know what it is. That’s the Human brain's intelligence.</p>

<p class=MsoNormal>Recently, Artificial intelligence has been widely used in
our lives, such as FaceID and ChatGPT.</p>

<p class=MsoNormal>AI encompasses various methods, and deep learning is one of
the most significant AI approaches, which involves processing data through
multiple layers of computation. It looks like the human brain's neural
structure. </p>

<p class=MsoNormal>AI was very popular in many fields,  especially Biology. 
CNN has a great performance in image recognition, and Bi-LSTM has better
performance in capturing <b>sequential</b> information. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In bioinformatics, numerous well-known AI models are
available. Recently.  PsiPred can predict secondary protein structures with a <b>neural
network. Given a sequence, alphafold can predict the 3D structure of proteins
with high accuracy. Due to the groundbreaking work, the creator of AlphaFold
received the Nobel Prize last year. </b></p>

<p class=MsoNormal><b>&nbsp;</b></p>

<p class=MsoNormal>In section 1 of my first project. We aim to develop a
domain-based deep learning model to represent the protein families and
superfamilies.</p>

<p class=MsoNormal>As we know, PSSM can present the evolutionary information of
the proteins. We aim to investigate whether PSSM can be utilized to make
predictions for protein superfamilies. </p>

<p class=MsoNormal>Another point is why I want to create a <b>domain-based</b> model.
A protein's complete function depends on the combination of different domains.
Each domain has its specific function.</p>

<p class=MsoNormal>For a protein with multiple domains, annotating a protein
with a single function is not ideal. </p>

<p class=MsoNormal>The full-length-based model will mislead in the study of
protein evolution.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Our workflow contains three main steps. <b>For data
preparation, we used the PSI-BLAST to generate the pssm and then unified the dimensions.
Additionally, data splitting involves randomly dividing</b> the data into three
sets: training data, a validation set, and a testing set. </p>

<p class=MsoNormal><b>During model training, we attempted to create different
LSTM structure models because a part of the PSSM is meaningless.  LSTM can also
capture global information from the PSSM.</b></p>

<p class=MsoNormal><b>&nbsp;</b></p>

<p class=MsoNormal><b>Here, we consider the superfamily BECR as an example.  BECR</b>
is one of the largest protein superfamilies with an assemblage of
metal-independent RNases.</p>

<p class=MsoNormal>Representative members of BECR <span style='font-family:
Wingdings'>à</span> <b>B</b>arnase, <b>E</b>ndoU, <b>C</b>olicin E5/D, and <b>R</b>elE.
</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For each family, 10 - 15 representative sequences were
randomly selected to generate PSSM profiles.</p>

<p class=MsoNormal>Number of profiles: 6984 (belong to 45 BECR super-families
and 399 other super-families).</p>

<p class=MsoNormal>80% <span style='font-family:Wingdings'>à</span> Training
data; 10% <span style='font-family:Wingdings'>à</span> Validation data; 10%<span
style='font-family:Wingdings'>à</span> Testing data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Each time of training, we can calculate the True positive,
False positive, False Negative, and True Negative, given the prediction
results. The Evaluation indicators, Including Recall, Precision, Accuracy, and
F1 score, can be used to evaluate the model's performance.  </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>In the final model, we found that when the epoch exceeds 30,
the Evaluation indicators become stable.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Finally, we obtained a model with an accuracy of 96% on the testing
data.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>We want to test the robustness of the model with three
independent families. And the accuracy is 67%.</p>

<div style='border:none;border-bottom:solid windowtext 1.0pt;padding:0cm 0cm 1.0pt 0cm'>

<p class=MsoNormal style='border:none;padding:0cm'>&nbsp;</p>

</div>

<p class=MsoNormal>In Section 2. We want to create a structure-based model.</p>

<p class=MsoNormal>As we are all aware, different proteins have different
structures. The function of proteins depends on their structure.</p>

<p class=MsoNormal>With AlphaFold, we can predict the structure from sequences.  
However, 3D information is not easily readable by a computer.  A contact map is
a 2D representation of the 3D structure. </p>

<p class=MsoNormal>Someone can identify the superfamily type after training
with many contact maps, but it is not easy. “Recognizing the contact map is
kind of like the cat-dog recognition problem.”</p>

<p class=MsoNormal>Input the kcontact map into the CNN model, and after
multiple convolutional layers, it can output the category of the superfamily.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Before inputting the CNN, we need to perform a matrix
transformation due to the outliers. Gaussian functions can smoothly transform
the distance to probability with range (0,1], especially when there are some
outliers.</p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Our
model has a good performance with an accuracy of 97% on testing data.</p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>Three
families were used to make a test as independent data, and the accuracy rate
is  66%.</p>

<div style='border:none;border-bottom:solid windowtext 1.0pt;padding:0cm 0cm 1.0pt 0cm'>

<p class=MsoNormal style='border:none;padding:0cm'>&nbsp;</p>

</div>

<p class=MsoNormal>In Section 3. We aim to develop a model that incorporates
both evolutionary and structural information. However,  a big challenge is how
to merge two datasets.</p>

<p class=MsoNormal>As we know, a colorful picture can be represented by three
channels: the red channel, the green channel, and the blue channel.  We can
therefore place PSSM and the contact map in two channels. However, PSSM and
contact map have different dimensions. A contact map is a self-self distance
matrix, and the dimension is LxL. PSSM refers to the specific scores for the
target 20 amino acids, with a dimension of L x 20. If we can convert the PSSM
to a square matrix, then it can continue with the channels. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Here, we use the cosine similarity transformation to convert
the PSSM to a square matrix. By comparing the similarity of different amino
acids in 20 vectors, we can get the cosine similarity of amino acids in a
protein.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The workflow is as follows. We generate two-channel datasets
with a Structure generated by AlphaFold and a PSSM generated by PSI-BLAST.
After training with multiple layers, it can report to us the superfamily type.</p>

<p class=MsoNormal>&nbsp;</p>

<ol style='margin-top:0cm' start=1 type=1>
 <li class=MsoNormal>Our model has a good performance with an accuracy of 99%
     on testing data.</li>
 <div style='border:none;border-bottom:solid windowtext 1.0pt;padding:0cm 0cm 1.0pt 0cm;
 margin-left:18.0pt;margin-right:0cm'>
 <li class=MsoNormal style='margin-left:18.0pt;border:none;padding:0cm'>Three
     families were used to make a test as independent data, and the accuracy
     rate is  86%.</li>
 </div>
</ol>

<p class=MsoNormal style='margin-left:36.0pt'>In brief, we developed two
models: a profile-based deep learning model and a profile-structure-based deep
learning model, specifically tailored for the BECR family.</p>

<ol style='margin-top:0cm' start=2 type=1>
 <ol style='margin-top:0cm' start=1 type=1>
  <li class=MsoNormal style='margin-left:54.0pt'>1. The accuracy of the three
      kinds of domain-based deep learning models with testing data is about
      98%. The domain-based deep learning model achieves its best accuracy with
      independent data, at 86%.</li>
  <li class=MsoNormal style='margin-left:54.0pt'>2. The model combined with
      PSSM and Contact map has the best performance in all three models. </li>
 </ol>
</ol>

<p class=MsoNormal style='margin-left:18.0pt'>=========================================================================Project
2<span lang=ZH-CN style='font-family:DengXian'>&#65306;</span>Hierarchical
clustering of highly divergent proteins</p>

<p class=MsoNormal style='margin-left:18.0pt'>Protein classification helps
establish sequence-function relationships.</p>

<p class=MsoNormal style='margin-left:18.0pt'>Traditional phylogenetic
approaches are effective for similar sequence families but struggle with highly
divergent families.</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<ol style='margin-top:0cm' start=1 type=1>
 <li class=MsoNormal style='margin-left:54.0pt'>With Alphafold, we can predict
     many structures. </li>
 <li class=MsoNormal style='margin-left:54.0pt'>DALI comparison of contact maps
     to evaluate structural similarity.</li>
</ol>

<p class=MsoNormal style='margin-left:36.0pt'>3. Conversion of similarity
scores into divergence scores.</p>

<p class=MsoNormal style='margin-left:36.0pt'>4. hierarchical clustering
analysis (HCA)</p>

<p class=MsoNormal style='margin-left:18.0pt'>               Ward’s </p>

<p class=MsoNormal style='margin-left:18.0pt'>              Ward.D2 methods</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>• Ward’s and Ward's D2 methods
showed the best performance in classification.</p>

<p class=MsoNormal style='margin-left:18.0pt'>• Structural clustering
effectively groups divergent protein families.</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>Thanks to all the members of Dr. Zhang's
lab and my committee members.</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

<p class=MsoNormal style='margin-left:18.0pt'>&nbsp;</p>

</div>

</body>

</html>
